### PR TITLE
Updating python container version to beam-master-20180518

### PR DESCRIPTION
**Pull request by angoenka**
Updating python worker container version to beam-master-20180518
------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

### CODE REVIEW

**Comment by angoenka** [comment]
R: @tvalentyn

File: sdks/python/apache_beam/runners/dataflow/internal/dependency.py [hunk 1]
```diff
@@ -78,10 +78,10 @@
 
 # Update this version to the next version whenever there is a change that will
 # require changes to legacy Dataflow worker execution environment.
-BEAM_CONTAINER_VERSION = 'beam-master-20180413'
+BEAM_CONTAINER_VERSION = 'beam-master-20180518'
 # Update this version to the next version whenever there is a change that
 # requires changes to SDK harness container or SDK harness launcher.
-BEAM_FNAPI_CONTAINER_VERSION = 'beam-master-20180413'
+BEAM_FNAPI_CONTAINER_VERSION = 'beam-master-20180518'
```
Comment (tvalentyn) [inline sdks/python/apache_beam/runners/dataflow/internal/dependency.py]: Please revert this line, we did not release FnAPI container.

**Comment by angoenka** [comment]
Run Python Dataflow ValidatesRunner

**Comment by angoenka** [comment]
Run Python Dataflow ValidatesRunner

**Comment by angoenka** [comment]
Done.
@jkff Please merge the change

**Comment by jkff** [comment]
Needs to be rebased now.

**Comment by angoenka** [comment]
This PR is not required anymore as the functionality moved to names.py and already have these changes PR-5251 https://github.com/apache/beam/pull/5251/commits/10e5c791c062a674c5593ff3a98d61a9d0882aa3

**Pull request closed without merging**